server/column_scope.cc

```diff
@@ -215,7 +215,8 @@ runIncrementalT(const std::vector<BoundSqlExpression> & exprs,
                     
                 size_t numRows = endOffset - startOffset;
 
-                Val results[exprs.size()];
```

Comment:
If we use the placement new technique, the ctor and dtor of `<typename Val> won't be called. Isn't that a risk?

Also, the unique_ptr pattern is also used on line 207 so if we do something I suggest we fix both.